kubeshop/testkube-dashboard
# Move error reports into the area where they were triggered

This PR...

## Changes
https://github.com/kubeshop/testkube/issues/2708
- moved error notifications triggered in forms such as creation and settings
- refactored Configuration card, adding ability for future refactor https://github.com/kubeshop/testkube/issues/3787

## Fixes

-

## How to test it
- create test/suite/executor etc
- try to make some error so notification appear
- edit some settings cards across the dashboard

## screenshots

-

## Checklist
